scripts/check_cargo_min_ver.sh

```diff
@@ -7,13 +7,14 @@ function usage() {
     echo
     echo "--help                    Print this message"
     echo "--rust-toolchain          The toolchain to check the version for with leading"
-    echo "--min-rust-version        Check toolchain version is >= to this version, default is 1.65"
+    echo "--min-rust-version        Check toolchain version is >= to this version, default is 1.67"
     echo
 }
```

Comment:
the usage function should be defined below so the help string can read the `MIN_RUST_VERSION` variable

Reply:
oh yeah, that's a mistake on my end